instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py

```diff
@@ -119,18 +129,18 @@ def response_hook(span: Span, params: typing.Union[
     HTTP_DURATION_HISTOGRAM_BUCKETS_OLD,
     _client_duration_attrs_new,
     _client_duration_attrs_old,
-    _filter_semconv_duration_attrs,
+    _filter_semconv_duration_attrs,  # type: ignore[reportUnknownVariableType]
     _get_schema_url,
     _OpenTelemetrySemanticConventionStability,
     _OpenTelemetryStabilitySignalType,
     _report_new,
     _report_old,
-    _set_http_host_client,
-    _set_http_method,
-    _set_http_net_peer_name_client,
-    _set_http_peer_port_client,
-    _set_http_url,
-    _set_status,
+    _set_http_host_client,  # type: ignore[reportUnknownVariableType]
```

Comment:
Could we avoid adding these `ignore` if `opentelemetry.instrumentation._semconv` had more type hints added to it first?